YoutubeExplode/Playlists/PlaylistController.cs

```diff
@@ -47,16 +47,65 @@ public PlaylistController(HttpClient http)
         var raw = await SendHttpRequestAsync(request, cancellationToken);
         var playlist = PlaylistExtractor.Create(raw);
 
-        if (!playlist.IsPlaylistAvailable())
+        if (!playlist.IsPlaylistDetailsAvailable())
+        {
+            //try getting info from the next endpoint if this one can't get the playlist
+            playlist = await GetPlaylistVideosAsync(playlistId,cancellationToken);
+        }
+
+        return playlist;
+    }
+
+    public async ValueTask<PlaylistExtractor> GetPlaylistVideosAsync(
```

Comment:
Can we use this for all playlists, including for getting metadata?

Reply:
I thought we could, but the problem I faced was that it didn't return the playlist description.